earth_enterprise/rpms/build.gradle

```diff
@@ -0,0 +1,138 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+    }
+}
+
+apply plugin: "nebula.ospackage"
+
+def env = System.getenv()
+def home = env['HOME']
```

Comment:
I can't see where you're using `home`, and I don't think the build should depend on the user's home directory.

Reply:
Oh, I was trying that out as an example - loading properties from commandline, env vars and from files.  Cleaning up!